napari/components/_viewer/model.py

```diff
@@ -217,30 +217,38 @@ def to_svg(self, file=None, canvas_shape=None):
             either a str or bytes object representing a path, or an object
             implementing the `os.PathLike` protocol. If passed the svg will be
             written to this file
-        canvas_shape : 2-tuple, optional
-            Shape of SVG canvas to be generated. If not specified, takes the
-            shape of the last two dimensions of the view
+        view_box : 4-tuple, optional
+            View box of SVG canvas to be generated specified as `min-x`,
+            `min-y`, `width` and `height`. If not specified, calculated
+            from the last two dimensions of the view.
 
         Returns
         ----------
         svg : string
             String with the svg specification of the currently viewed layers
         """
 
-        if canvas_shape is None:
-            canvas_shape = self._calc_max_shape()[-2:]
+        if view_box is None:
```

Comment:
you define `min_shape`, `max_shape`, and `range` when `view_box` is `None` but do not define them when `view_box` is not `None`

Reply:
why overwrite the built-in `range` when that doesn't really capture what it represents here anyways? i think `shape` makes more sense

Reply:
fixed